src/core/prodfactor.jl

```diff
@@ -99,8 +99,8 @@ function prodfactor(wind::Wind, latitude::Real, longitude::Real, time_steps_per_
 
     if all(length(a) > 0 for a in [wind.temperature_celsius, wind.pressure_atmospheres, wind.wind_direction_degrees,
                                    wind.wind_meters_per_sec])
-        push!(resources, [wind.temperature_celsius, wind.pressure_atmospheres, wind.wind_direction_degrees,
-                          wind.wind_meters_per_sec])
+        push!(resources, hcat([wind.temperature_celsius, wind.pressure_atmospheres, wind.wind_meters_per_sec, wind.wind_direction_degrees]...))
+        resources = hcat(resources...)
```

Comment:
it appears that in both of these lines some values are concatenated with `hcat` then immediately unpacked with `...`. Looks unnecessary.

Reply:
Removed the surplus hcat when pushing wind resources to resources[] in https://github.com/NREL/REopt.jl/commit/c13a75042f1421ef0834058133cef6d0d98f229f

Reply:
Thanks Bhavesh!